tools/gfx/vulkan/vk-fence.cpp

```diff
@@ -40,6 +44,30 @@ Result FenceImpl::init(const IFence::Desc& desc)
     createInfo.pNext = &timelineCreateInfo;
     createInfo.flags = 0;
 
+#if SLANG_WINDOWS_FAMILY
+    VkExportSemaphoreWin32HandleInfoKHR exportSemaphoreWin32HandleInfoKHR;
+#endif
+    VkExportSemaphoreCreateInfoKHR exportSemaphoreCreateInfo;
+    if (desc.isShared)
+    {
+#if SLANG_WINDOWS_FAMILY
+        exportSemaphoreWin32HandleInfoKHR.sType = VK_STRUCTURE_TYPE_EXPORT_SEMAPHORE_WIN32_HANDLE_INFO_KHR;
+        exportSemaphoreWin32HandleInfoKHR.pNext = NULL;
```

Comment:
I would be tempted to make this `pNext` equal to `timelineCreateInfo.pNext` (currently also `nullptr`). This will make it still work should we add any other next structs above. (same for the one below)

Reply:
Done